### PR TITLE
AC: add adaptor and metrics for attribute classification models

### PR DESCRIPTION
**Pull request by tlsdmstn56-2**
* Add attribute_classification adapter convert outputs of attribute classification models to `ContainerPrediction` which contains multiple `ClassificationPrediction`s with their scores.
* Add precision, recall, accuracy for attribute classification models. Each metric takes `ContainerAnnotation` with `ClassificationAnnotation`and `ContainerPrediction` with `ClassificationPrediction` and outputs its metric scores per attribute and their mean optionally.
* Update REAMD.md

### CODE REVIEW

**Comment by eaidova** [comment]
jenkins ok to test

File: tools/accuracy_checker/accuracy_checker/adapters/attribute_classification.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+"""
+Copyright (c) 2019 Intel Corporation
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/adapters/attribute_classification.py:2]: please apply correct license years for all modified files
```suggestion
Copyright (c) 2019-2020 Intel Corporation
```

File: tools/accuracy_checker/accuracy_checker/metrics/attribute_classification.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    def evaluate(self, annotations, predictions):
+        pass
+
+    def submit_all(self, annotations, predictions):
+        return self.evaluate(annotations, predictions)
+
+    def compute_annotation_prediction_dict(self, annotations, predictions):
+        if self.is_annotation_prediction_dict_computed:
+            return
+        self.is_annotation_prediction_dict_computed = True
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/attribute_classification.py:65]: please add reset method for reset annotation_prediction_dict, it is need for metric reusage in calibration process

File: tools/accuracy_checker/accuracy_checker/metrics/attribute_classification.py [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from sklearn.metrics import accuracy_score, precision_score, recall_score
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/attribute_classification.py:18]: in some platform like yocto, sklearn may be unavailable, so you metric can not work for it. If it is not important for you, then stay it as it, otherwise please rewrite without sklearn usage.
